backend/src/models/Application.ts

```diff
@@ -0,0 +1,46 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+import { applicationStatusSchema } from "./ApplicationStatus";
+
+const applicationSchema = new Schema({
+  _id: {
+    type: String,
+    unique: true,
+    required: true,
+  },
```

Comment:
The `_id` field would be automatically generated by mongoose so I think we can remove this portion

Reply:
Noted